### PR TITLE
TE-10233: CS core module run as separate ones

### PR DESCRIPTION
**Pull request by olhalivitchuk**
## PR Description

[Code sniffer: Core module run as separate ones](https://spryker.atlassian.net/browse/TE-10233)

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: src/GitHook/Command/FileCommand/PreCommit/CodeStyleSniff/CodeStyleSniffConfiguration.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+{
+    /**
+     * @var string
+     */
+    protected const MODULE_CONFIG_TOOL_KEY = 'code-sniffer';
+
+    /**
+     * @var string
+     */
+    protected const CORE_MODULE_PATH_REGEX = '/.*\/vendor\/spryker\/.*\/Bundles\/.*\/src\//';
```
Comment (dereuromark) [inline src/GitHook/Command/FileCommand/PreCommit/CodeStyleSniff/CodeStyleSniffConfiguration.php]: Do we need this one? It also doesnt seem to work for spryker-shop or other Namespaces

Comment (olhalivitchuk) [reply src/GitHook/Command/FileCommand/PreCommit/CodeStyleSniff/CodeStyleSniffConfiguration.php]: It matches changed module to get its `tooling.yml`.
This regex works with any `spryker/*` package

Comment (dereuromark) [reply src/GitHook/Command/FileCommand/PreCommit/CodeStyleSniff/CodeStyleSniffConfiguration.php]: Well, I guess any nonsplit
my main concern is the hardcoding of nonsplit Bundles/ stuff that is only visible and useful Spryker internally (never externally on split).

But fair enough for now then.

Comment (dereuromark) [reply src/GitHook/Command/FileCommand/PreCommit/CodeStyleSniff/CodeStyleSniffConfiguration.php]:
```suggestion
    protected const CORE_MODULE_PATH_REGEX = '/.*\/vendor\/spryker\/.+\/Bundles\/.+\/src\//';
```
I dont see how it is useful to use any when it must be at least a non empty string.

Comment (dereuromark) [reply src/GitHook/Command/FileCommand/PreCommit/CodeStyleSniff/CodeStyleSniffConfiguration.php]: Also, use # as regexp delimiter, then you dont need to escape normal forward slash.

**Review by dereuromark — APPROVED** [review]

**Review by stereomon — APPROVED** [review]
Looks good to me but I miss some tests for it. I'll not block this because of missing tests...

**Pull request merged**